### PR TITLE
[doc fix] fix doc of auc

### PR DESCRIPTION
**Pull request by wangzhen38**
添加ins_tag_weight属性对应的中文文档

### CODE REVIEW

**Review by jzhang533 — COMMENTED** [review]
请对照着`paddle.static.auc`的英文文档进行修改。

**Comment by wangzhen38** [comment]
> 请对照着`paddle.static.auc`的英文文档进行修改。

done

File: docs/api/paddle/static/auc_cn.rst [hunk 1]
```diff
@@ -25,12 +25,13 @@ auc
 参数
 ::::::::::::
 
-    - **input** (Tensor|LoDTensor) - 数据类型为 float32、float64。浮点二维变量，值的范围为[0,1]。每一行降序排列。该输入为网络预测值的输入。
-    - **label** (Tensor|LoDTensor) - 数据类型为 int32、int64。二维整型变量，为训练数据的标签。
+    - **input** (Tensor) - 数据类型为 float32、float64。浮点二维变量，值的范围为[0,1]。每一行降序排列。该输入为网络预测值的输入。
```
Comment (jzhang533) [inline docs/api/paddle/static/auc_cn.rst]: 英文文档中的这句："Typically, this Tensor indicates the probability of each label."， 在这里没有体现出来。

File: docs/api/paddle/static/auc_cn.rst [hunk 2]
```diff
@@ -25,12 +25,13 @@ auc
...
+    - **label** (Tensor) - 数据类型为 int32、int64。二维整型变量，为训练数据的标签。
```
Comment (jzhang533) [inline docs/api/paddle/static/auc_cn.rst]: 与英文文档不一致。英文版本：
A 2D int Tensor indicating the label of the training data. The height is batch size and width is always 1. A Tensor with type int32,int64.

File: docs/api/paddle/static/auc_cn.rst (hunk 1 above)
Comment (wangzhen38) [reply docs/api/paddle/static/auc_cn.rst]: done

File: docs/api/paddle/static/auc_cn.rst (hunk 2 above)
Comment (wangzhen38) [reply docs/api/paddle/static/auc_cn.rst]: done

**Review by jzhang533 — APPROVED** [review]
LGTM

**Review by yaoxuefeng6 — APPROVED** [review]
